python/tvm/auto_scheduler/auto_schedule.py

```diff
@@ -136,9 +159,14 @@ def __init__(
 
         if isinstance(runner, str):
             if runner == "local":
-                runner = LocalRunner()
+                runner = LocalRunner(working_dir=self.temp_working_dir)
             else:
                 raise ValueError("Invalid runner: " + runner)
+
+        elif isinstance(runner, RPCRunner):
+            rpc_kwargs = runner.kwargs
+            rpc_kwargs["working_dir"] = self.temp_working_dir
```

Comment:
How this argument be used later? I can not find any logic to handle this parameter...

Reply:
https://github.com/apache/tvm/pull/7053/files#diff-5e828f80da7fdc456523468e7f69c1617d8a88867500f68998567fbd6f95a1d7R183